src/main/java/org/popcraft/chunky/Chunky.java

```diff
@@ -138,6 +142,13 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
         return Collections.emptyMap();
     }
 
+    private void startEnabledWatchdogs() {
+        //TODO: Check which ones are enabled and only start those
+        WatchdogManager manager = new WatchdogManager(this);
+        manager.registerWatchdog(new PlayerWatchdog());
+        Bukkit.getScheduler().runTaskTimer(this, manager, 0, 20);
```

Comment:
Not sure if it's necessary to run an extra task -- status could be checked in the generation tasks themselves

Reply:
The reason I did it in a separate task is because I can't really think of a way to restart the generation tasks once the tasks are stopped in a nice way. If the goal is to prevent threads being created an destroyed, I can probably put them to sleep instead.